test/e2e/apps/disruption.go

```diff
@@ -215,6 +201,38 @@ var _ = SIGDescribe("DisruptionController", func() {
 			}
 		})
 	}
+
+	var _ = Describe("UpdatingPodDisruptionBudgets", func() {
```

Comment:
I don't think we need to create a new `Describe`. Let's just add new `It`s inside the existing `SIGDescribe("DisruptionController", ...`.

Reply:
ok removed